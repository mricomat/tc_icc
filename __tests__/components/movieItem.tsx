import React from 'react';
import { render } from '@testing-library/react-native';

import MovieItem from 'src/components/movieItem';

describe('test Input', () => {
  test('it renders correctly', () => {
    const input = render(<MovieItem />);
    expect(input).toMatchSnapshot();
  });

  test('it renders correctly with title and date', () => {
    const input = render(<MovieItem title={'test'} date={'titleDate'} />);
    expect(input).toMatchSnapshot();
  });
});
