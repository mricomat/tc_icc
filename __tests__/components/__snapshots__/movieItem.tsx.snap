// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Input it renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "borderRadius": 8,
        "flex": 1,
        "minHeight": 140,
      },
      Object {},
    ]
  }
>
  <Image
    source={
      Object {
        "uri": undefined,
      }
    }
    style={
      Object {
        "borderRadius": 8,
        "height": "100%",
        "opacity": 0.2,
        "position": "absolute",
        "width": "100%",
      }
    }
  />
  <View
    style={
      Object {
        "flexDirection": "row",
        "height": "100%",
        "padding": 10,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "borderRadius": 6,
            "height": "100%",
            "width": 80,
          },
          Object {
            "backgroundColor": "#DADADA",
            "margin": 10,
            "position": "absolute",
          },
        ]
      }
    />
    <Image
      source={
        Object {
          "uri": undefined,
        }
      }
      style={
        Object {
          "borderRadius": 6,
          "height": "100%",
          "width": 80,
        }
      }
    />
    <View
      style={
        Object {
          "flex": 1,
          "paddingHorizontal": 6,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontSize": 18,
            "fontWeight": "600",
          }
        }
      />
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontSize": 16,
            "fontWeight": "400",
            "marginTop": 8,
          }
        }
      />
    </View>
  </View>
</View>
`;

exports[`test Input it renders correctly with title and date 1`] = `
<View
  style={
    Array [
      Object {
        "borderRadius": 8,
        "flex": 1,
        "minHeight": 140,
      },
      Object {},
    ]
  }
>
  <Image
    source={
      Object {
        "uri": undefined,
      }
    }
    style={
      Object {
        "borderRadius": 8,
        "height": "100%",
        "opacity": 0.2,
        "position": "absolute",
        "width": "100%",
      }
    }
  />
  <View
    style={
      Object {
        "flexDirection": "row",
        "height": "100%",
        "padding": 10,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "borderRadius": 6,
            "height": "100%",
            "width": 80,
          },
          Object {
            "backgroundColor": "#DADADA",
            "margin": 10,
            "position": "absolute",
          },
        ]
      }
    />
    <Image
      source={
        Object {
          "uri": undefined,
        }
      }
      style={
        Object {
          "borderRadius": 6,
          "height": "100%",
          "width": 80,
        }
      }
    />
    <View
      style={
        Object {
          "flex": 1,
          "paddingHorizontal": 6,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontSize": 18,
            "fontWeight": "600",
          }
        }
      >
        test
      </Text>
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontSize": 16,
            "fontWeight": "400",
            "marginTop": 8,
          }
        }
      >
        titleDate
      </Text>
    </View>
  </View>
</View>
`;
